### PR TITLE
EE for AAP2 Windows

### PR DESCRIPTION
**Pull request by bbethell-1**

### CODE REVIEW

File: tools/execution_environments/ee-multicloud-public/requirements.yml [hunk 1]
```diff
@@ -50,3 +50,9 @@ collections:
 
 # openstacksdk>=1.0.0
 - name: openstack.cloud
+
+- name: ansible.posix
+  version: 1.3.0
```
Comment (fridim) [inline tools/execution_environments/ee-multicloud-public/requirements.yml:55]: Please do not pin the version here

File: tools/execution_environments/ee-multicloud-public/requirements.yml [hunk 2]
```diff
@@ -50,3 +50,9 @@ collections:
...
+
+- name: ansible.workshops
+  version: 1.0.18
```
Comment (fridim) [inline tools/execution_environments/ee-multicloud-public/requirements.yml:58]: same here.

Also please sort alphabetically.

**Review by fridim — CHANGES_REQUESTED** [review]

File: tools/execution_environments/ee-multicloud-public/requirements.yml (hunk 1 above)
Comment (bbethell-1) [reply tools/execution_environments/ee-multicloud-public/requirements.yml:55]: @miteshget

File: tools/execution_environments/ee-multicloud-public/requirements.yml (hunk 2 above)
Comment (bbethell-1) [reply tools/execution_environments/ee-multicloud-public/requirements.yml:58]: @miteshget

Comment (fridim) [reply tools/execution_environments/ee-multicloud-public/requirements.yml:58]: We won't integrate `ansible.workshops` into the default supported EE.

This is catalog item specific. Plus this collection pulls different version of aws collection:

```diff
    version: 6.4.0
+    version: 3.1.1
```

You will see it in the diff in this PR.

I suggest you start a branch to maintain your own EE for this Catalog Item.

Comment (fridim) [reply tools/execution_environments/ee-multicloud-public/requirements.yml:58]: Actually, i think you should create a new folder:

*  `tools/execution_environments/ee-aap2`

And update it there.  You'll need a quay repo as well

Comment (miteshget) [reply tools/execution_environments/ee-multicloud-public/requirements.yml]: Resolved all above query, crated new directory tools/execution_environments/ee-ansible-workshop with all of the requirements for new ee

**Review by fridim — APPROVED** [review]

**Pull request merged**